docs/api/tasks-get-task-by-title

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+---
+
+# Get task by title
+
+Returns an array of  [`task`](task) objects that contain only the tasks specified by the `title` parameter, if they exist.
+
+## URL
+
+```shell
+
+{server_url}/task/{title}
```

Comment:
```suggestion
{server_url}/task?title=<title value>
```

I think this needs to be a query parameter. Where you have `title` is where the unique task ID would go.